### ISSUE TITLE
Start with semantic versionen.

### ISSUE THREAD

**Issue by GordianDziwis** [Scaseco/facete3#12]
First move fast then: https://semver.org/

**Comment by Aklakan** [comment]
I agree we should stick to semver as much as possible - but I do not think it would be good to have e.g. the vaadin interface versioned separately. It is closely coupled with the facete api so when the API changes, it should be in the same project so we can easily see if things break.
So I'd give maintainability within our small team a priority over having nicer looking versions.


So there should be one generic Vaadin UI in the facete codebase - and maybe forks for domain specific demos which can have independent versions. But as we all know, activity on these demo forks will usually cease quickly.


Just look at jena: They release 50+ modules at once - versioning those modules independently would be hell.
